app/src/interfaces/useCritterApi.interface.ts

```diff
@@ -31,12 +31,11 @@ export interface IMarkingPostData {
 
 export interface ICapturePostData {
   capture_id: string;
-  capture_timestamp: string;
-  capture_date?: string;
-  capture_time?: string;
-  release_timestamp: string;
-  release_date?: string;
-  release_time?: string;
+  capture_date: string;
+  capture_method_id: string | null;
```

Comment:
optional nit: maybe move this up so its not between capture_date and capture_time?

Reply:
fixed